src/unix/fs.c

```diff
@@ -229,9 +229,46 @@ static ssize_t uv__fs_futime(uv_fs_t* req) {
 #endif
 }
 
+static ssize_t uv__fs_access(uv_fs_t * req) {
+#ifdef __MVS__
+  int r = 0;
+
+  if (req->flags | F_OK)
```

Comment:
The logic looks incorrect to me.

If F_OK == 0, it's equivalent to `if (req->flags)` which seems wrong; the branch is taken when `req->flags != F_OK`.

If it's non-zero, then it should be `if (req->flags & F_OK)`, otherwise the branch is always taken.